recognition-server/server.py

```diff
@@ -14,7 +18,11 @@ def hello():
     if 'image' in request.files:
         f = request.files['image']
         ext = '.' + f.filename.split('.')[-1]
+        filePath = 'tmp/' + str(uuid.uuid4()) + ext
         f.save('tmp/' + str(uuid.uuid4()) + ext)
+
+        os.system('python Recognition.py')
```

Comment:
as you have imported **Recognition** at top
you can call this as 

vehicle_id = Recognition.recognise(filepath)